.github/workflows/nebula-publish.yml

```diff
@@ -13,6 +13,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
+      - run: |
+            git config --global user.name "Netflix OSS Maintainers"
+            git config --global user.email "netflixoss@netflix.com"
```

Comment:
nit: could you pl update the details as follows?
1. user.name - Mantis OSS Maintainers
2. user.email — mantis-oss-dev@googlegroups.com

Reply:
will do